projects/angular/src/forms/common/abstract-container.ts

```diff
@@ -47,15 +46,22 @@ export abstract class ClrAbstractContainer implements DynamicWrapper, OnDestroy,
     );
 
     this.subscriptions.push(
-      this.ngControlService.controlChanges.subscribe(control => {
-        this.control = control;
-      }),
-      this.ngControlService.secondaryControlChanges.subscribe(control => {
-        this.secondaryControl = control;
+      this.ngControlService.controlChanges.subscribe(controls => {
+        this.controls = controls;
       })
     );
   }
 
+  get control(): NgControl {
```

Comment:
We want to preserve this so we don't need to rework each and every current component.